src/app/api/handle-chapter-request/route.ts

```diff
@@ -0,0 +1,86 @@
+import { NextRequest, NextResponse } from "next/server";
+import {
+  HandleChapterRequest,
+  HandleChapterRequestResponse,
+} from "./route.schema";
+import { prisma } from "@server/db/client";
+
+export const POST = async (request: NextRequest) => {
```

Comment:
This route should be protected - i.e, only an admin can manage it. We export a decorator `withRole()` from `@server/decorator/index` that handles this.